src/Castle.Core.Tests/PublicApiTestCase.cs

```diff
@@ -31,6 +31,7 @@ namespace Castle
 	///   the assertion will obviously fail if someone changes the public API without also including the change to the ref/*.cs files
 	/// </summary>
 	[TestFixture]
+	[System.Runtime.Versioning.SupportedOSPlatform("windows")]
```

Comment:
(Looking at this change while reviewing this PR commit by commit:)

~~The commit message claims this will lead to the code only getting executed on Windows. I thought this attribute only affects compilation warnings; in order to actually filter a test by platform, wouldn't we typically use NUnit's `[Platform]` attribute?~~

(Ah, you're going to replace with `[Platform(...)]` in the next commit. I suppose you could squash those two commits to clean things up a little, but I'm fine with it either way.)